src/framework/chart/LineChart.jsx

```diff
@@ -36,6 +37,42 @@ export default class LineChart extends Component {
     this.container = this.lineChartSvg.append('g');
 
     this.renderChart(this.props, true);
+
+    this.lineChartSvg.on('mousemove', () => {
+      if (d3.event.target.isSameNode(this.refs.lineChartSvg)) {
```

Comment:
Good catch. Then this attempt of mine apparently doesn't work as intended. The problem is that if I remove the if and you move the mouse over a horizontal line the x position will be taken from that element which leads to the first two dots being displayed.

So I will need to change the way I get the x position of the mouse.